HW04.Rmd

```diff
@@ -69,7 +95,7 @@ Assume that:
 * Fall = October-December
 
 ```{r season}
-
+## Not really sure how to do this one, do we filter for each individual month then add those numbers up and create another table? 

```

Comment:
I was a little confused here as well.  I used the function fct_collapse to collapse factor levels into manually defined groups for each season.  You can read more about it here (https://forcats.tidyverse.org/reference/fct_collapse.html).

mutate(season = fct_collapse(.f = month, Winter = c("Jan", "Feb", "Mar"), .......continue on for other seasons / months.

Reply:
Thanks for the recommendation! I checked this out and found it very helpful. I looked at your answer for this part of the assignment after reading this and the question really seemed designed for this fct_collapse function.